tests/sync/test_fuzz_guard_integration.py

```
@@ -0,0 +1,311 @@
+    return HttpResponse("OK")
+
+
+class TestFuzzGuardEndToEnd(TestCase):
+    """End-to-end integration tests for blocked requests."""
+
+    def setUp(self):
+        self.client = Client()
+
+    def test_normal_request_passes(self):
+        """Normal request should not trigger fuzz guard."""
+        initial_count = FuzzAttempt.objects.count()
```

Ruff (0.15.1)

[error] Local variable `initial_count` is assigned to but never used

Remove assignment to unused variable `initial_count`

(F841)